### DISCUSSION TITLE
Pre-website submitted (submitted as a GitHub pr) icons are not showing up on profile.

### DISCUSSION THREAD

**Discussion by OzturkAlperen** [elrumo/macOS_Big_Sur_icons_replacements#1181] [Help with the website] [answered]
So I have quite a lot icons submitted with a pull request. Obviously they don't show up on user profile. Is there way to 'convert' those icons? Or will there be a way in the future? 

For example: 
<img width="1270" alt="Ekran Resmi 2021-05-24 20 38 31" src="https://user-images.githubusercontent.com/63194133/119386332-0d762c80-bcd0-11eb-9d78-56d82637d498.png">
<img width="492" alt="Ekran Resmi 2021-05-24 20 38 16" src="https://user-images.githubusercontent.com/63194133/119386337-0f3ff000-bcd0-11eb-8ff6-13922aea7dd4.png">

Thanks in advance.

**Comment by elrumo** [comment, marked as answer]
At the moment it's something only I can do, but I'm working to give more management features to users. In the meantime, I've just linked them to your new account.

**Comment by elrumo** [comment]
Also, I've noticed that some of your icons seem to be larger than the rest, I'd recommend you resize them and re-upload them again.

**Reply by OzturkAlperen** [reply]
Yes, I have noticed that too. This happened after the 'hack'. Some of my icons gone completely. I have to reupload them too. But I can't see an option delete/update existing icons. Thanks for the help by the way.
